packages/libs/error-stack/src/fmt/mod.rs

```diff
@@ -166,187 +170,30 @@ use owo_colors::{OwoColorize, Stream::Stdout, Style as OwOStyle};
-/// by **any** hook, they will be rendered in the order: `C`, `B`, `A`.
-///
-/// # Example
-///
-/// ```rust
-/// # // we only test with Rust 1.65, which means that `render()` is unused on earlier version
-/// # #![cfg_attr(not(rust_1_65), allow(dead_code, unused_variables, unused_imports))]
-/// use std::io::{Error, ErrorKind};
-///
-/// use error_stack::{fmt::Emit, Report};
-///
-/// struct Warning(&'static str);
-/// struct ErrorCode(u64);
-/// struct Suggestion(&'static str);
-/// struct Secret(&'static str);
-///
-/// Report::install_debug_hook::<ErrorCode>(|ErrorCode(val), _| {
-///     vec![Emit::next(format!("Error Code: {val}"))]
-/// });
-/// Report::install_debug_hook::<Suggestion>(|Suggestion(val), _| {
-///     vec![Emit::defer(format!("Suggestion: {val}"))]
-/// });
-/// Report::install_debug_hook::<Warning>(|Warning(val), _| {
-///     vec![Emit::next("Abnormal program execution detected"), Emit::next(format!("Warning: {val}"))]
-/// });
-/// Report::install_debug_hook::<Secret>(|_, _| vec![]);
-///
-/// let report = Report::new(Error::from(ErrorKind::InvalidInput))
-///     .attach(ErrorCode(404))
-///     .attach(Suggestion("Do you have a connection to the internet?"))
-///     .attach(ErrorCode(405))
-///     .attach(Warning("Unable to determine environment"))
-///     .attach(Secret("pssst, don't tell anyone else c;"))
-///     .attach(Suggestion("Execute the program from the fish shell"))
-///     .attach(ErrorCode(501))
-///     .attach(Suggestion("Try better next time!"));
-///
-/// # owo_colors::set_override(true);
-/// # fn render(value: String) -> String {
-/// #     let backtrace = regex::Regex::new(r"Backtrace No\. (\d+)\n(?:  .*\n)*  .*").unwrap();
-/// #     let backtrace_info = regex::Regex::new(r"backtrace with (\d+) frames \((\d+)\)").unwrap();
-/// #
-/// #     let value = backtrace.replace_all(&value, "Backtrace No. $1\n  [redacted]");
-/// #     let value = backtrace_info.replace_all(value.as_ref(), "backtrace with [n] frames ($2)");
-/// #
-/// #     ansi_to_html::convert_escaped(value.as_ref()).unwrap()
-/// # }
-/// #
-/// # #[cfg(rust_1_65)]
-/// # expect_test::expect_file![concat!(env!("CARGO_MANIFEST_DIR"), "/tests/snapshots/doc/fmt__emit.snap")].assert_eq(&render(format!("{report:?}")));
-/// #
-/// println!("{report:?}");
-/// ```
-///
-/// <pre>
-#[doc = include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/tests/snapshots/doc/fmt__emit.snap"))]
-/// </pre>
 #[derive(Debug, Clone, PartialEq, Eq)]
 #[must_use]
 #[non_exhaustive]
-pub enum Emit {
+pub(crate) enum Emit {
     /// Line is going to be emitted after all immediate lines have been emitted from the current
     /// stack.
     /// This means that deferred lines will always be last in a group.
-    #[cfg_attr(not(any(feature = "std", feature = "spantrace")), allow(dead_code))]
+    // Reason: We could in theory dead-code, but that would hurt readability in the rendering
+    // process.
```

Comment:
A word is missing here

Reply:
fixed in [7640562](https://github.com/hashintel/hash/pull/1008/commits/76405629a6351a476bb7a4be3a1591fdc26ebca2)